vars/deploy-on-openstack.yml

```diff
@@ -23,7 +23,6 @@ deploy_on_openstack_ses_repos_per_imagename:
     - SES5-update
 deploy_on_openstack_osh_repos_per_imagename:
   openSUSE-Leap-15.0:
-    - Leap15develtools
```

Comment:
I don't think this is used anywhere else so we should remove it from common vars